Sources/MapboxMaps/Gestures/GestureHandlers/PitchGestureHandler.swift

```diff
@@ -45,16 +45,10 @@ internal final class PitchGestureHandler: GestureHandler, UIGestureRecognizerDel
             }
 
             let translation = gestureRecognizer.translation(in: view)
-            let translationAngle = angleOfLine(from: .zero, to: translation)
-
-            // If the angle between the touch locations is less than the maximum
-            // AND the translation angle is more than 60 degrees, update the pitch.
-            if touchAngleIsLessThanMaximum(for: gestureRecognizer), abs(translationAngle) > 60 {
```

Comment:
Is there a chance that single-touch gestures are now being recognized since we don't call `touchAngleIsLessThanMaximum` any more, where we checked that number of touches was 2?

Reply:
That should be prevented by lines 12 and 13. The check in `touchAngleIsLessThanMaximum` is more relevant *after* the gesture begins to recognize as a safeguard to avoid index out of bounds errors when obtaining touches on lines 26 and 27.